setup.py

```diff
@@ -1,37 +1,65 @@
 import os.path
 from setuptools import setup, find_packages
 
+
 def read(*path_elements):
-    return "\n\n" + file(os.path.join(*path_elements)).read()
+    return "\n\n" + open(os.path.join(*path_elements), 'rt').read()
```

Comment:
Might as well use `with open(...) as f: return ... + f.read()` to avoid ResourceWarnings that Python 3 shows for implicitly closed files.